src/MSBuild.UnitTests/TerminalLogger_Tests.cs

```diff
@@ -38,6 +38,7 @@ public class TerminalLogger_Tests : IEventSource, IDisposable
 
         private readonly string _projectFile = NativeMethods.IsUnixLike ? "/src/project.proj" : @"C:\src\project.proj";
         private readonly string _projectFile2 = NativeMethods.IsUnixLike ? "/src/project2.proj" : @"C:\src\project2.proj";
+        private readonly string _projectFileWithForeignSymbols = NativeMethods.IsUnixLike ? "/src/проектТерминал/本地化.proj" : @"C:\src\проектТерминал\本地化.proj";
```

Comment:
Should we add any of the GB18030 PUA characters here?

Reply:
done